simplewebrtc.js

```diff
@@ -231,6 +231,7 @@ function SimpleWebRTC(opts) {
         });
     });
     this.webrtc.on('localScreenStopped', function (stream) {
+        self.emit('localScreenRemoved');
```

Comment:
in `stopScreenShare`, there's an event emitted `unshareScreen` - does that accomplish what you're trying to add, or does this serve another purpose?